btr-web/btr-main-app/pages/staffRequestView.vue

```diff
@@ -0,0 +1,347 @@
+<script setup lang="ts">
+
+import { useOmitIndividual } from '@/stores/omit-individual'
+import { RequestStatusesE } from '~/enums/request-statuses-e'
+
+const omitIndividual = useOmitIndividual()
+const MAX_COMMENT_SIZE = 2000
+const { activeRequest, activeComments, staffComment, changeState } = storeToRefs(omitIndividual)
+
+const t = useNuxtApp().$i18n.t
+
+const loading = ref(true)
+const identifier = ref('')
+
+const statusKeys = Object.keys(RequestStatusesE)
+const statusFilterOptions = []
+for (let i = 0; i < statusKeys.length; i++) {
+  statusFilterOptions.push({ value: statusKeys[i], label: RequestStatusesE[statusKeys[i]] })
+}
+
+const formatDate = function (date: string, includeTime?: boolean) {
```

Comment:
Just checking for this and 'numDaysBetweenNow' - Is there a common date function already defined that we could use? I don't remember what we have in BTR but generally we have a common date util with things like this

Reply:
There isn't one in btr I was looking for it but I didn't see any date utils, happy to start one with this though